src/sentry/releasehealth/metrics.py

```diff
@@ -150,3 +158,185 @@ def _compute_crash_free_rate(data: Dict[str, float]) -> Optional[float]:
         crash_free_rate = 100 * max(0.0, crash_free_rate)
 
         return crash_free_rate
+
+    def get_release_adoption(
+        self,
+        project_releases: Sequence[Tuple[ProjectId, ReleaseName]],
+        environments: Optional[Sequence[EnvironmentName]] = None,
+        now: Optional[datetime] = None,
+        org_id: Optional[OrganizationId] = None,
+    ) -> ReleaseHealthBackend.ReleasesAdoption:
```

Comment:
Not a fan of the type subclassing.. I would just move it outside the class.. Surely calling this function elsewhere in the code base will make me require that type